### DISCUSSION TITLE
Welcome to popcorn-time-api Discussions!

### DISCUSSION THREAD

**Discussion by DEADSEC-SECURITY** [DEADSEC-SECURITY/popcorn-time-api#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by perryaa** [comment]
Since the Ukrainian war began, I have not been able to open or see any TV shows from the ru.tk link shown in the settings. I can see movies since the server seems to be in Mexico. So, has the war stopped the server in russia? Is there a new link for TV shows I can enter in the settings of popcorn-time?

**Reply by DEADSEC-SECURITY** [reply]
@perryaa sorry for the late response, your issue has been answered and a new version has been released with the new domain.
